examples/calibration/plot_calibration.py

```diff
@@ -35,7 +36,7 @@
 
 from sklearn.datasets import make_blobs
 from sklearn.naive_bayes import GaussianNB
-from sklearn.metrics import brier_score_loss
+from sklearn.metrics import calibration_loss
```

Comment:
Maybe we should rename it to `calibration_error` to be consistent with `ECE` / `MCE` and further ease discoverability via google?